ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp.j2

```diff
@@ -7,4 +7,10 @@ router_bgp:
 
 {%     include 'network_services/router-bgp-vlan-aware-bundles.j2' %}
 
+{%     if default_vrf.enabled %}
+
+{%         include 'network_services/default-vrf.j2' %}
```

Comment:
rename the template to `router-bgp-default-vrf.j2`

Reply:
Sure.